api-guidelines/rest/http/headers/rules/must-send-user-agent-header.md

```diff
@@ -0,0 +1,15 @@
+---
+id: R000074
+---
+
+# MUST send `User-Agent` header
+
+API providers must be able to easily identify which client is calling their API.
+The `User-Agent` header saves the API provider manual work, e.g. retrieving this information from the token or reading the access log.
```

Comment:
access log reading is still needed, but the User-Agent value is directly visible in the access log, which is not the case for e.g. `client_id` JWT claim as part of the OAuth2 Bearer Token.